src/shims/foreign_items.rs

```diff
@@ -591,14 +594,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             }
             // underscore case for windows
             "_ldexp" | "ldexp" => {
```

Comment:
Could this be made to support `scalbn` itself?

Reply:
Sure, why not.